frontend/pages/hosts/ManageHostsPage/_styles.scss

```diff
@@ -204,7 +204,7 @@
               tr {
                 th {
                   .last-col-header-with-tip {
-                    min-width: 90px;
+                    min-width: 155px;
```

Comment:
This re-introduces the spacing problem with tooltips at the x-end of the table:
<img width="367" alt="Screenshot 2023-09-25 at 2 10 01 PM" src="https://github.com/fleetdm/fleet/assets/61553566/a9cd268a-2749-4d9a-9fce-1f27047946f2">

Is it necessary?  If so, I think it's fine, since the new TooltipWrapper will solve this by the end of the week, but ideal not to reintroduce it if possible.

Reply:
Don't know which is the lesser of 2 evils, the white space at the end if it's the last column, or the cut off tooltip if it's only 1 row of results?

Either way, I don't think this bug is worth delaying a release.

Reply:
> since the new TooltipWrapper will solve this by the end of the week

@jacobshandling looks like we're not trying to get this in so lets just go with the fix with the new wrapper!

Reply:
Sounds good. I'll go ahead and close this PR.